### PR TITLE
Added the possibility to consume more than one good at once

### PR DESCRIPTION
**Pull request by CrsiX**
This feature request allows a consumer to consume more than one good at a time. When no argument is given, it will assume it's one ice/water/drink/pizza.

### CODE REVIEW

File: src/commands/consume.py [hunk 1]
```diff
@@ -1,29 +1,62 @@
...
+
+
+def get_amount_helper(msg, name) -> int:
+	args = parseArgs(msg, [ARG_INT], [False], "/{} [amount]".format(name))
+	if isinstance(args[0], int):
+		if args[0] <= 0:
+			msg.reply_text("Amount must be a positive integer")
+			return 0
+		elif args[0] > 10:
+			msg.reply_text("Amount max. is 10!")
```
Comment (M4GNV5) [inline src/commands/consume.py:13]: What amount refers to is not quite obvious to the user without knowing the code. Maybe something like `You cannot consume a negative amount` and `You cannot consume more than 10 {}s at once`

File: src/commands/consume.py [hunk 2]
```diff
@@ -1,29 +1,62 @@
...
 def water(bot, update):
-	user = getOrCreateUser(update.message.from_user)
-	createTransaction(user, -50, "water")
-	update.message.reply_text(random.choice(hydrationMessages), disable_notification=True)
+	num = get_amount_helper(update.message, "water")
+	if num > 0:
+		user = getOrCreateUser(update.message.from_user)
+		createTransaction(user, -50 * num, "water x{}".format(num))
+		answer = random.choice(hydrationMessages)
+		update.message.reply_text(answer[0].format(answer[1]*num), disable_notification=True)
```
Comment (M4GNV5) [inline src/commands/consume.py:46]: spacing between the operators please. `answer[1] * num`, or even better: `num * answer[1]`

File: src/commands/consume.py [hunk 3]
```diff
@@ -1,29 +1,62 @@
...
 
 def pizza(bot, update):
-	user = getOrCreateUser(update.message.from_user)
-	createTransaction(user, -200, "pizza")
-	update.message.reply_text("Buon appetito! 🍕", disable_notification=True)
+	num = get_amount_helper(update.message, "pizza")
+	if num > 0:
+		user = getOrCreateUser(update.message.from_user)
+		createTransaction(user, -200 * num, "pizza x{}".format(num))
+		update.message.reply_text("Buon appetito! {}".format('🍕'*num), disable_notification=True)
```
Comment (M4GNV5) [inline src/commands/consume.py:54]: same here

File: src/commands/consume.py [hunk 4]
```diff
@@ -1,29 +1,62 @@
...
 
 def ice(bot, update):
-	user = getOrCreateUser(update.message.from_user)
-	createTransaction(user, -50, "ice")
-	update.message.reply_text("Have a sweet one! 🚅", disable_notification=True)
+	num = get_amount_helper(update.message, "ice")
+	if num > 0:
+		user = getOrCreateUser(update.message.from_user)
+		createTransaction(user, -50 * num, "ice x{}".format(num))
+		update.message.reply_text("Have a sweet one! {}".format('🚅'*num), disable_notification=True)
```
Comment (M4GNV5) [inline src/commands/consume.py:62]: and here

**Review by M4GNV5 — CHANGES_REQUESTED** [review]
Overall a good request, but some minor changes have to be made before merging.

File: src/commands/consume.py [hunk 5]
```diff
@@ -1,29 +1,62 @@
 from state import getOrCreateUser, createTransaction
 import random
+from args import parseArgs, ARG_INT
+
+
+def get_amount_helper(msg, name) -> int:
```
[the remark this answers is not in the document]
Comment (CrsiX) [reply src/commands/consume.py:6]: Indeed, I had the code of another project in my mind, when I created this feature ;)

File: src/commands/consume.py (hunk 1 above)
Comment (CrsiX) [reply src/commands/consume.py:13]: I agree, I fixed it

**Comment by CrsiX** [comment]
I've checked it again and resolved your comments as well as a little bug. Thanks for your support.

**Review by M4GNV5 — APPROVED** [review]

**Pull request merged**